constants/gfx_constants.asm

```diff
@@ -1,3 +1,13 @@
+; * LCD enabled
+; * Window tile map at $9C00
+; * Window display enabled
+; * BG and window tile data at $8800
+; * BG tile map at $9800
+; * 8x8 OBJ size
+; * OBJ display enabled
+; * BG display enabled
+DEF LCDC_DEFAULT EQU LCDC_ON | LCDC_WIN_9C00 | LCDC_WIN_ON | LCDC_BLOCK21 | LCDC_BG_9800 | LCDC_OBJ_8 | LCDC_OBJ_ON | LCDC_BG_ON
```

Comment:
Please move this to the bottom of ram_constants.asm, to match pokegold and pokecrystal, since this value is associated with the RAM address `rLCDC`.